src/test/codegen/inline-hint.rs

```diff
@@ -0,0 +1,30 @@
+// Checks that closures, constructors, and shims except
+// for a drop glue receive inline hint by default.
+//
+// compile-flags: -Cno-prepopulate-passes -Zsymbol-mangling-version=v0
+// revisions: CLOSURE TUPLE CLONE DROP
```

Comment:
Why are revisions used, when all of them are compiled with the same settings?

Reply:
To avoid checking the order in which those functions are emitted.

Reply:
[`CHECK-DAG`](https://www.llvm.org/docs/CommandGuide/FileCheck.html#the-check-dag-directive) is a more natural approach here, I feel.

Reply:
I removed revisions and reordered the checks.